src/screens/Booking/components/PassengerForm/Form.tsx

```diff
@@ -0,0 +1,101 @@
+import {useFormik} from 'formik';
+import React from 'react';
+import * as Yup from 'yup';
+import {t} from '@translations';
+import {useDispatch} from 'react-redux';
+import {AppDispatch} from '@store/store';
+import {buy} from '@modules/ticket';
+import {View, Text, TouchableOpacity} from 'react-native';
+import {InputField} from '@components';
+import styles from './Form.styles';
+
+interface FormProps {
+  flightId: string;
+}
+
+const Form = ({flightId}: FormProps) => {
+  const initialValues = {
+    name: '',
+    surname: '',
+    passport: '',
+  };
+
+  const validationSchema = Yup.object().shape({
+    name: Yup.string().required(`${t.form.validations.required}`),
+    surname: Yup.string().required(`${t.form.validations.required}`),
+    passport: Yup.string()
+      .max(9, `${t.form.validations.passportTooLong}`)
+      .min(9, `${t.form.validations.passportTooShort}`)
+      .required(`${t.form.validations.required}`),
+  });
```

Comment:
yup schema can be defined outside the Form component and for the passport schema definition
it's best to use yup string().length

```
passport: Yup.string()
      .length(9, `${message here}`)
      .required(`${t.form.validations.required}`),
```